packages/material-ui/src/Modal/Modal.d.ts

```diff
@@ -29,6 +29,26 @@ export interface ModalProps
   open: boolean;
 }
 
+/**
+ * Modal is a lower-level construct that is leveraged by the following components:
+ *
+ * -   [Dialog](https://material-ui.com/api/dialog/)
```

Comment:
```suggestion
 * -  [Dialog](https://material-ui.com/api/dialog/)
```

Reply:
I don't know where this is coming from. But vscode trims it anyway so I'm not investigating further until this becomes a proble.